code/game/objects/machinery/doors/airlock_types.dm

```diff
@@ -550,7 +550,7 @@
 	icon = 'icons/obj/doors/mainship/medidoor.dmi'
 	req_access = list(ACCESS_MARINE_RESEARCH)
 
-/obj/machinery/door/airlock/mainship/rebel/research
+/obj/machinery/door/airlock/mainship/research/rebel
```

Comment:
ended up changing this back as I believe if I left it as /rebel/research, it would mean I'd also have to state the icon and name.  Which seems like a bit of unnecessary bloat, considering it was also done this way.  May be wrong, though.